src/NCPushButton.cc

```diff
@@ -123,10 +124,11 @@ void NCPushButton::wRedraw()
 		  NC::CENTER );
 }
 
+// pressing the button
 void NCPushButton::activate()
 {
-	// focus the button
-	setKeyboardFocus();
-	// and fake pressing the RETURN key
-	::ungetch(KEY_RETURN);
+    // send an activation event for this widget
```

Comment:
That `setKeyboardFocus()` is no longer neccessary? This might be a subtle change of behaviour. Please check with our QA people if their tests rely on this.

Reply:
That `setKeyboardFocus()` is actually already called in the REST plugin ([here](https://github.com/libyui/libyui-rest-api/blob/master/src/YHttpWidgetsActionHandler.cc#L85)) for both Qt and Ncurses frontends.

And the Qt event handler also does not set it so it should behave the same way.

Reply:
OK.